config/routes.rb

```diff
@@ -19,6 +19,7 @@
   resources :nodes, only: [:index, :show, :destroy] do
     collection do
       post :bootstrap
+      post :assign_roles
```

Comment:
you mean undescores in the url endpoint? not sure what is best here TBH, but we also have `/users/sign_in`, `/users/sign_out`